src/constants/r2Prefixes.ts

```diff
@@ -23,3 +23,15 @@ export const VIRTUAL_DIRS: Record<string, Set<string>> = {
       .map(([key]) => key.substring(12) + '/')
```

Comment:
I think it's not trivial that here you're doing key.substring(12) because that's the length of `nodejs/docs` I would probably prefer something more explicit